src/System.Net.Security/src/System/Net/SecurityStatus.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net
+{
+    internal enum SecurityStatus
```

Comment:
Is this really needed? Since it's an internal type (it's actually the Windows SDK `SECURITY_STATUS`) I would expect it'll disappear and manifest itself as Exceptions thrown by the PAL layer.